app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if min_number_of_users_who_made_edits > 1 %}
+              More than one user has edited this service. The last user to edit this page was
```

Comment:
Should be `The last user to edit this service` - not page.

Reply:
Absolutely, changing this now